scala-package/pom.xml

```diff
@@ -112,6 +112,44 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>scala-2.11</id>
+    </profile>
+
+    <profile>
+      <id>scala-2.12</id>
+      <properties>
+        <scala.version>2.12.4</scala.version>
+        <scala.binary.version>2.12</scala.binary.version>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
why do we need this plugin?

Reply:
To ensure that no Scala 2.10 or 2.11 versioned dependencies are pulled in.